configure.cmd

```diff
@@ -118,6 +133,8 @@ REM Parse command line parameters
             REM explicitly set VS version for project generator
             if /I "%2" == "-VS_VER" (
                 set VS_VERSION=%3
+                SHIFT
+                SHIFT
```

Comment:
Nice command, didn't knew about it. Maybe `SHIFT /n 2` instead of two commands?

Reply:
Removed it entirely